sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
Sorry for this late reply. I've just updated the commit candidate. I need to test.